src/tests/postprocessor.rs

```diff
@@ -0,0 +1,29 @@
+use std::{
+    env::current_dir, fs::{self, Permissions}, os::unix::fs::PermissionsExt
+};
+
+use crate::compiler::postprocess::{PostProcessor, PostProcessorInput, PostProcessorOutput};
+
+fn create_fake_binary(processor: PostProcessor) {
+    let body = if cfg!(unix) {
+        "#!/usr/bin/env bash\nexit 0"
+    } else {
+        panic!("this test is not available for non-unix platforms")
```

Comment:
Can we simply exit gracefully on windows?

Reply:
why? it will stop the other tests from running. this way, only the ones depending on this function will fail

Reply:
right now it was added the windows support https://github.com/amber-lang/amber/pull/501

Reply:
It doesn't even work for me on Linux:
```
hwalters@Ghostwheel ~/git/amber (postprocess) 
$ amber echo.ab -
thread 'main' panicked at src/compiler/postprocess.rs:54:66:
Couldn't get shfmt's stdin
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
read /dev/stdin: input/output error
```
And yes, I do have `shfmt` installed:
```
hwalters@Ghostwheel ~/git/amber (postprocess) 
$ which shfmt 
/usr/bin/shfmt
```
Can we fix this before I check it on Windows?

Reply:
However, I suspect it will work ok, as Git Bash scripts can be run with shebang lines just like Linux and MacOS Bash.

Reply:
> Git Bash scripts can be run with shebang lines just like Linux and MacOS Bash.

that's only for git bash. you are forgetting if they run it via cmd or ps, it simply won't work in that case

Reply:
i've rewritten it to depend on that all default postprocessors are installed and available on the test machine